src/content/get-started/use-a-model.mdx

```diff
@@ -1,15 +1,15 @@
 ---
-title: "Try a Model"
-metaTitle: "Try a Model"
-metaDescription: "Try a Model with the DeepSparse Engine to deploy for faster and cheaper inference on CPUs"
+title: "Use a Model"
+metaTitle: "Use a Model"
+metaDescription: "Use a Model with DeepSparse to deploy for faster and cheaper inference on CPUs"
```

Comment:
Since DeepSparse describes the suite of components, are you saying one can deploy with the whole suite? Are all four components doing the deployment?